### ISSUE TITLE
Security Hub tests broken

### ISSUE THREAD

**Issue by s0enke** [superwerker/superwerker#86]
```
======================================================================
114 | ERROR: test_security_hub_cannot_be_disabled_in_member_account (tests.MyTestCase)
115 | ----------------------------------------------------------------------
116 | Traceback (most recent call last):
117 | File "/codebuild/output/src713553801/src/tests/tests.py", line 135, in test_security_hub_cannot_be_disabled_in_member_account
118 | scp_test_role_creds = log_archive_account_sts.assume_role(
119 | File "/root/.pyenv/versions/3.8.1/lib/python3.8/site-packages/botocore/client.py", line 316, in _api_call
120 | return self._make_api_call(operation_name, kwargs)
121 | File "/root/.pyenv/versions/3.8.1/lib/python3.8/site-packages/botocore/client.py", line 635, in _make_api_call
122 | raise error_class(parsed_response, operation_name)
123 | botocore.exceptions.ClientError: An error occurred (AccessDenied) when calling the AssumeRole operation: User: arn:aws:sts::534317841728:assumed-role/AWSControlTowerExecution/superwerkertest is not authorized to perform: sts:AssumeRole on resource: arn:aws:iam::A:role/SuperWerkerScpTestRole
```

```
133 | ======================================================================
134 | FAIL: test_securityhub_enabled_with_delegated_admin_in_core_and_enrolled_accounts (tests.MyTestCase)
135 | ----------------------------------------------------------------------
136 | Traceback (most recent call last):
137 | File "/codebuild/output/src713553801/src/tests/tests.py", line 93, in test_securityhub_enabled_with_delegated_admin_in_core_and_enrolled_accounts
138 | self.assertCountEqual(expected_members, actual_members)
139 | AssertionError: Element counts were not equal:
140 | First has 1, Second has 0:  'A'
141 | First has 1, Second has 0:  'B'
142

**Comment by s0enke** [comment]
first fix: https://github.com/superwerker/superwerker/commit/2616a1feea1b9e68c005fba82a2908b939b711b8

**Comment by s0enke** [comment]
fixed with 72adad6bac7ba44ac026fb313b332874c48a20df and d29396a3b6050e1055fa84de77bc1c7835d05773
